CODEOWNERS

```
@@ -1 +1 @@
-* @btat @LucasSaintarbor @sunilarjun
\ No newline at end of file
+* @btat @LucasSaintarbor @sunilarjun @pmkovar
```

```suggestion
* @btat @LucasSaintarbor @pmkovar @sunilarjun
```

nit: let's keep it alphabetical

---

Thanks for catching Billy! Updated in this commit:https://github.com/rancher/rancher-docs/pull/1802/commits/cc307bb6b795be9b70c2a47fe74d2fff6a61d9e4.